### PR TITLE
Add llama370b_instruct_v1

### PR DESCRIPTION
**Pull request by ErebusBat**
Adds Llama 3 Instruct model

https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-meta.html#model-parameters-meta-request-response

This PR bumps the version by two as it build on my other PR #3

### CODE REVIEW

**Comment by AAlvAAro** [comment]
Hi @ErebusBat looks like you'll have to fix conflicts for this PR. Also would you mind trying to add base64 so this issue can be closed and we can link to the PRs. Thanks a lot!

**Comment by ErebusBat** [comment]
I will rebase to fix conflicts, and look into #5.

File: lib/amazon_bedrock/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RubyAmazonBedrock
-  VERSION = "0.2.3"
+  VERSION = "0.2.5"
```
Comment (AAlvAAro) [inline lib/amazon_bedrock/version.rb:4]: Let's keep it to 0.2.4 version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [0.2.4] - 2024-04-25
+
+- Support the use of AWS Named Profiles for authentication
```
Comment (AAlvAAro) [inline CHANGELOG.md:3]: Hi @ErebusBat let's add all your changes to the same version, please add the following notes to the CHANGELOG 

- Support for llama370b_instruct_v1
- Explicitly add base64 gem to dependencies

File: README.md [hunk 3]
```diff
@@ -63,16 +63,14 @@ client = RubyAmazonBedrock::Client.new(
...
-## With Configuration
+### AWS Named Profiles
 
-```ruby
-RubyAmazonBedrock.configure do |config|
-  config.region = ENV.fetch('AWS_REGION', nil)
-  config.access_key_id = ENV.fetch('AWS_ACCESS_KEY_ID', nil)
-  config.secret_access_key = ENV.fetch('AWS_SECRET_ACCESS_KEY', nil)
-end
+You can also use [AWS Named Profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-format-profile) by passing the `profile` keywoard argument. When using a named profile, specyfing the `region`, `access_key_id` and `access_token` won't be required.
```
Comment (AAlvAAro) [inline README.md:68]: Don't remove the With Configuration documentation, you can add the AWS Named Profiles note below that.

File: spec/bedrock_runtime/client_spec.rb [hunk 4]
```diff
@@ -8,7 +8,8 @@
     described_class.new(
       region: ENV.fetch('AWS_REGION', nil),
       access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID', nil),
-      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY', nil)
+      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY', nil),
+      profile: ENV.fetch('AWS_PROFILE', nil)
```
Comment (AAlvAAro) [inline spec/bedrock_runtime/client_spec.rb:12]: Please add a new test scenario where the client instantiation is done without the `profile` so both cases are covered.

**Review by AAlvAAro — CHANGES_REQUESTED** [review]
Hi @ErebusBat thanks for the PR, I left a few comments and please make sure `rspec` and `rubocop` pass. Thanks!

**Review by ErebusBat — COMMENTED** [review]
@AAlvAAro My apologies for the delay on addressing these issues.

It looks like some issues on this PR should have been addressed on #3... See comments.

I did update `.env.sample` so that once people copy it to `.env` they can just run the existing specs as I had some issues getting it up and running.

Both `rspec` and `rubocop` are passing on my machine now

**Review by AAlvAAro — APPROVED** [review]

**Pull request merged**